scan-command/src/main/java/io/ballerina/scan/utils/ScanUtils.java

```diff
@@ -276,6 +301,242 @@ private static void unzipReportResources(InputStream source, File target) throws
         }
     }
 
+    /**
+     * Returns the {@link ScanTomlFile} representation of a local/remote Scan.toml file.
+     *
+     * @param project      Ballerina project
+     * @param outputStream print stream
+     * @return an in-memory representation of the Scan.toml file
+     */
+    public static ScanTomlFile loadScanTomlConfigurations(Project project, PrintStream outputStream) {
+        Path targetDir = project.targetDir();
+
+        if (project.kind().equals(ProjectKind.BUILD_PROJECT)) {
+            if (project.currentPackage().ballerinaToml().isPresent()) {
```

Comment:
Doesn't the project kind being BUILD_PROJECT guarantee that there is a ballerinaToml

Reply:
Since the return type of the ballerinaToml() method is optional, will invert conditions with an early return